src/winter_onboarding_2021/fleet_management_service/handlers/cab.clj

```diff
@@ -6,6 +6,15 @@
             [winter-onboarding-2021.fleet-management-service.config :as config]
             [winter-onboarding-2021.fleet-management-service.specs :as specs]))
 
+(defn flash-msg [msg success?]
```

Comment:
Shouldn't this be private?

Reply:
This might be needed in some other handlers as well. Like for user etc. 
cc- @shhivam